cd . # really sure that is just because of insufficient power supply for external drives?
pytest-3 -s test_make_dog_cat_connection.py::test_query